aten/src/ATen/native/TensorShape.cpp

```diff
@@ -909,7 +909,15 @@ Tensor view(const Tensor& self, IntArrayRef size) {
     "not compatible with input tensor's size and stride (at least one dimension"
     " spans across two contiguous subspaces). Use .reshape(...) instead.");
   auto stride_value = *stride;
-  auto self_ = self.clone();
+  Tensor self_;
+  if (self.is_quantized()) {
+    TORCH_CHECK(self.qscheme() == kPerTensorAffine,
+                "Only PerTensorAffine quantization is supported right now");
+    self_ = at::_empty_affine_quantized({0}, self.options(), self.q_scale(),
+                                        self.q_zero_point());
+  } else {
+    self_ = at::empty({0}, self.options());
```

Comment:
this is still more expensive - we create an empty storage here and the go on to replace it. You probably need to drop down one level lower and create tensor with explicitly provided storage instead